stance/main.py

```diff
@@ -13,20 +11,26 @@
     args = parser.parse_args()
 
     # Find the front and profile skeletons
+    print("Fitting images")
```

Comment:
We can put in an issue to change these to proper logger messages if we care enough later